src/Action/Iterator.php

```diff
@@ -70,7 +70,8 @@ public function like($field, $value)
                 return false;
             }
 
-            $clean_value = trim(trim($value), '%');
+            $value = trim($value);
+            $clean_value = trim($value, '%');
             // the row field exists check the position of the "%"(s)
             switch ($value) {
```

Comment:
This PR is fine. But what is actually the purpose of `switch` (vs. `if`) when all cases are expressions (i.e. not values for which the `switch` is designed)?